src/components/Receiving/LineDetails.js

```diff
@@ -61,22 +65,11 @@ const LineDetails = ({
       </div>
     ),
     'itemStatus': () => (
-      <div className={css.fieldWrapper}>
-        <Select
-          fullWidth
-          selectClass={css.itemStatusField}
-          value={ITEM_STATUS.inProcess}
-        >
-          {Object.keys(ITEM_STATUS).map((key) => (
-            <FormattedMessage
-              id={`ui-orders.receiving.itemStatus.${key}`}
-              key={key}
-            >
-              {(message) => <option value={ITEM_STATUS[key]}>{message}</option>}
-            </FormattedMessage>
-          ))}
-        </Select>
-      </div>
+      <SelectItemStatus
+        statuses={RECEIVE_ITEM_STATUSES}
+        value={ITEM_STATUS[STATUS_IN_PROCESS]}
+        selectClass={css.itemStatusField}
```

Comment:
what is the purpose of passing this class in every place? I think it should be managed inside of component

Reply:
This property is passed only in one place, unfortunately this code was in previous version and I'm not sure to delete it.

Reply:
moved the property in common component